src/core/public/application/ui/app_not_found_screen.tsx

```diff
@@ -48,12 +48,28 @@ export const AppNotFound = () => (
             </h2>
           }
           body={
-            <p>
-              <FormattedMessage
-                id="core.application.appNotFound.pageDescription"
-                defaultMessage="No application was found at this URL. Try going back or choosing an app from the menu."
+            <>
+              <p>
+                <FormattedMessage
+                  id="core.application.appNotFound.pageDescription"
+                  defaultMessage="No application was found at this URL. Try going back or choosing an app from the menu."
+                />
+              </p>
+              <EuiEmptyPrompt
+                actions={
+                  <EuiButton
+                    onClick={onClickGoHome}
+                    fill
+                    data-test-subj="createVisualizationPromptButton"
+                  >
+                    <FormattedMessage
+                      id="visualize.listing.createNew.createButtonLabel"
```

Comment:
```suggestion
                    data-test-subj="noAppDefaultPageButton"
                  >
                    <FormattedMessage
                      id="core.application.listing.appNotFound.defaultPageButton"
```

Looks like you copied over the ID's from elsewhere.